src/Get.tsx

```diff
@@ -214,8 +238,17 @@ function Get<TData = any, TError = any>(props: GetProps<TData, TError>) {
   return (
     <RestfulReactConsumer>
       {contextProps => (
-        <RestfulReactProvider {...contextProps} base={`${contextProps.base}${props.path}`}>
-          <ContextlessGet {...contextProps} {...props} />
+        <RestfulReactProvider
+          {...contextProps}
+          base={props.absolute ? contextProps.base : `${contextProps.base}/${normalizeUrlPath(props.path)}`}
+        >
+          <ContextlessGet
+            {...contextProps}
+            {...props}
+            originalBase={
```

Comment:
This statement deals with the manual overriding of the base prop. See [this test](https://github.com/contiamo/restful-react/pull/33/files#diff-ecba994a57db0df28ccfaccfaba2a1e0R280) that covers it.